src/app.js

```diff
@@ -16,10 +19,37 @@ app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
 
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
-
+const options = {
```

Comment:
In the dev branch, these options are moved to variables.js 
While merging your branch, just make sure to remove them from here and get the options from the variables.js file